src/test/scala/com/microsoft/hyperspace/index/HybridScanTest.scala

```diff
@@ -252,37 +270,38 @@ class HybridScanTest extends QueryTest with HyperspaceSuite {
               assert(children.last.isInstanceOf[ShuffleExchangeExec])
               assert(bucketSpec.numBuckets === 200)
               p
-            case p @ FileSourceScanExec(_, _, _, _, _, dataFilters, _) =>
+            case p @ FileSourceScanExec(_, _, _, partitionFilters, _, dataFilters, _) =>
               // Check filter pushed down properly.
               assert(
-                dataFilters.toString.contains(" >= 2000)") && dataFilters.toString.contains(
-                  " <= 4000)"))
+                (dataFilters.toString.contains(" >= 2000)") &&
+                  dataFilters.toString.contains(" <= 4000)")) ||
```

Comment:
Once you start adding more conditions in `assert` - especially `||` testing different objects `dataFilters` and `partitionFilters`, it's hard to reason which condition is met. Can you separate these into different `assert`s?